crux-bench/cloudformation.yaml

```diff
@@ -144,7 +144,40 @@ Resources:
         - 'FARGATE'
       NetworkMode: 'awsvpc'
       ContainerDefinitions:
+        - Name: 'zookeeper-container'
+          Cpu: 1024
```

Comment:
this seems high - can you confirm what zookeeper/kafka are actually using during a bench run?